GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/CarList.cs

```diff
@@ -0,0 +1,39 @@
+﻿public static class CarList
+{
+    public static readonly Car CarSedan = new Car("carSedan", 500, 0, false);
+    public static readonly Car CarTruck = new Car("carTruck", 600, 0, false);
+    public static readonly Car CarJeep = new Car("carJeep", 650, 0, true);
+    public static readonly Car CarKart = new Car("carKart", 1000, 0, true);
+
+    // return car object by name
+    public static Car GetCarByName(string carName)
```

Comment:
You don't need this method. Add a property to Car so that you can get the name by doing _carObject.name

Reply:
In this method, I am getting the car object by car name.